tensorboard/plugins/core/core_plugin.py

```diff
@@ -193,12 +194,38 @@ def _serve_environment(self, request):
                     "creation_time": experiment_metadata.creation_time,
                 }
             )
+        if self._include_debug_info:
+            environment["debug"] = {
+                "version": version.VERSION,
```

Comment:
Maybe worth including `version` unconditionally at the top level?  That seems safe, and it's more structured than the rest, so we could exempt it from the "human consumption only" constraint in case we do want to render it in the frontend.